integration/operator_test.go

```diff
@@ -252,27 +252,31 @@ func operatorSetup() error {
 		return microerror.Mask(err)
 	}
 
-	if err := writeAWSResourceValues(); err != nil {
-		return microerror.Maskf(err, "unexpected error writing aws-resource-lab values file")
+	err := writeAWSResourceValues()
+	if err != nil {
+		return microerror.Maskf(err, "writing aws-resource-lab values file")
 	}
 
-	if err := runCmd("helm registry install quay.io/giantswarm/aws-resource-lab-chart:stable -- -n aws-resource-lab --values " + awsOperatorValuesFile); err != nil {
```

Comment:
I think it makes sense to separate function calls and conditionals, especially when the function calls are bigger and more complex.